contracts/PriceOracle/PriceOracle.sol

```diff
@@ -201,10 +192,24 @@ contract PriceOracle is Ownable2Step {
      */
     function validateTokenConfig(TokenConfig memory config) internal view {
         if (config.cToken == address(0)) revert MissingCTokenAddress();
-        // Dual check of field being present and being non-zero
-        if (config.baseUnit == 0) revert InvalidBaseUnit();
         if (config.priceFeed == address(0)) revert InvalidPriceFeed(config.priceFeed);
         // Check if duplicate configs were submitted for the same cToken
         if (tokenConfigs[config.cToken].cToken != address(0)) revert DuplicateConfig(config.cToken);
+        validateDecimals(config.priceFeed, config.underlyingAssetDecimals);
     }
+
+    /**
+     * @notice Validates the combination of price feed decimals and the underlying asset decimals in the config
+     * @param priceFeed The price feed the decimals need to be validated for
+     * @param underlyingAssetDecimals The underlying asset decimals set in the config
+     */
+     function validateDecimals(address priceFeed, uint8 underlyingAssetDecimals) internal view {
+        // Check underlyingAssetDecimals exists and non-zero
+        if (underlyingAssetDecimals <= 0) revert InvalidUnderlyingAssetDecimals();
```

Comment:
```suggestion
        if (underlyingAssetDecimals == 0) revert InvalidUnderlyingAssetDecimals();
```

The `<` is redundant as `underlyingAssetDecimals` is an unsigned integer hence will never be negative.

Reply:
Fixed in the latest commit